elide-spring/elide-spring-boot-autoconfigure/src/test/java/com/yahoo/elide/spring/tests/DynamicConfigTest.java

```diff
@@ -0,0 +1,129 @@
+                                                attr("highScore", null),
+                                                attr("name", "SerenaWilliams")
+                                        )
+                                )
+                        ).toJSON())
+                )
+                .statusCode(HttpStatus.SC_OK);
+    }
+    @SqlMergeMode(SqlMergeMode.MergeMode.MERGE)
+    @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD,
+            statements = "INSERT INTO PlayerStats (name,countryId,createdOn) VALUES\n"
+                    + "\t\t('SerenaWilliams','1','2000-10-01');"
+                    + "INSERT INTO PlayerCountry (id,isoCode) VALUES\n"
+                    + "\t\t('1','USA');")
+    @Sql(executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD,
+            statements = "DELETE FROM PlayerStats; DELETE FROM PlayerCountry;")
+    @Test
+    public void jsonApiPostTestView() {
+        given()
+                .contentType(JsonApiController.JSON_API_CONTENT_TYPE)
+                .body(
+                        datum(
+                                resource(
+                                        type("playerStats"),
+                                        id("SaniaMirza"),
+                                        attributes(
+                                                attr("countryId", "1"),
+                                                attr("createdOn", "2002-03-01T04:00Z"),
+                                                attr("highScore", null)
+                                        )
+                                )
+                        )
+                )
+                .when()
+                .post("/json/playerStats")
+                .then()
+                .body(equalTo(datum(
+                        resource(
+                                type("playerStats"),
+                                id("SaniaMirza"),
+                                attributes(
+                                        attr("countryId", "1"),
+                                        attr("createdOn", "2002-03-01T04:00Z"),
+                                        attr("highScore", null)
+                                )
+                        )
+                ).toJSON()))
+                .statusCode(HttpStatus.SC_CREATED);
+    }
+    @SqlMergeMode(SqlMergeMode.MergeMode.MERGE)
+    @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD,
+            statements = "INSERT INTO PlayerStats (name,countryId,createdOn) VALUES\n"
+                    + "\t\t('SaniaMirza','2','2000-10-01');"
+                    + "INSERT INTO PlayerStats (name,countryId,createdOn) VALUES\n"
+                    + "\t\t('SerenaWilliams','1','2000-10-01');"
+                    + "INSERT INTO PlayerCountry (id,isoCode) VALUES\n"
+                    + "\t\t('2','IND');"
+                    + "INSERT INTO PlayerCountry (id,isoCode) VALUES\n"
```

Comment:
Since this statement is common for all test cases, we can move it up at class level.